PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -1978,6 +1978,8 @@ function Get-SddcDiagnosticInfo
                 }
 
                 $NodeSystemRootPath = Invoke-Command -ComputerName $using:NodeName { $env:SystemRoot }
+                $NodeMinidumpPath = Invoke-Command -ComputerName $using:NodeName { (Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\CrashControl').MinidumpDir }
+                $NodeLiveKernelReportsPath = Invoke-Command -ComputerName $using:NodeName { (Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\CrashControl').LiveKernelReportsPath }

```

Comment:
stylistically, i'd suggest just making these two paths what they need to be here; i.e., if null, set them to default. then we don't need ifs wrapping different ways to construct the adminsharepath.